packages/components/src/palette-edit/styles.js

```diff
@@ -41,7 +41,8 @@ export const NameInputControl = styled( InputControl )`
 `;
 
 export const PaletteItem = styled( View )`
-	padding: 3px 0 3px ${ space( 3 ) };
+	padding-block: 3px;
+	${ rtl( { paddingLeft: space( 3 ), paddingRight: 0 } ) };
```

Comment:
Could we just use `padding-inline-start`?

Reply:
Yes 👍  Start seems like a better solution. I will update the PR later today.